### ISSUE TITLE
Softmax operator not found

### ISSUE THREAD

**Issue by AbhinandanBedi** [azamsharp/TensorFlowToCoreML#1]
After running the **_inspect_pb.py_** file, a _**text_summary.txt**_ file was generated but I didn't get any Softmax operator at the end of the file. Can anyone figure out why?

![screen shot 2018-05-21 at 12 54 57 pm](https://user-images.githubusercontent.com/20858369/40295338-91d98a22-5cf6-11e8-86c3-0cae044cac92.png)
